packages/token-bridge-sdk/src/util/graph.ts

```diff
@@ -224,6 +216,7 @@ export const getTokenWithdrawals = async (
       ethBlockNum: BigNumber.from(ethBlockNum),
       callvalue: BigNumber.from(callvalue)
     } as L2ToL1EventResult
+    const tokenAddress = `0x${data.slice(34,74)}`;
```

Comment:
Not sure if this math works on hex. I think it'd be better to use `ethers.utils.hexDataSlice` so we can just use byte offsets. Also could we get a test to verify that this works as intended